addons/mrp/static/src/views/mrp_production_list/mrp_production_list_view.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import { listView } from "@web/views/list/list_view";
+import { ListController } from "@web/views/list/list_controller";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { registry } from "@web/core/registry";
+
+export class MrpProductionListController extends ListController {
+
+    async beforeExecuteActionButton(clickParams) {
+        if (clickParams.name === "action_cancel") {
+            const proceed = await new Promise((resolve) => {
+                this.dialogService.add(ConfirmationDialog, {
+                    body: this.env._t("Are you sure you want to cancel the manufacturing order(s) ?"),
+                    cancel: () => resolve(false),
+                    close: () => resolve(false),
```

Comment:
I don't think this key is used
```suggestion
```
That said, is it not better to do that server side instead of adding a js_class ?

Reply:
removed